ci-operator/config/opendatahub-io/odh-dashboard/opendatahub-io-odh-dashboard-main.yaml

```diff
@@ -39,6 +39,13 @@ tests:
     env:
       IMAGE_REPO: odh-dashboard
     workflow: opendatahub-io-ci-image-mirror
+- as: odh-dashboard-s390x-build
```

Comment:
Instead of just getting this in every PR we should create a cron job to run each day